app/models/spree/adjustment.rb

```diff
@@ -70,6 +70,7 @@ class Adjustment < ApplicationRecord
     scope :credit, -> { where('amount < 0') }
     scope :return_authorization, -> { where(originator_type: "Spree::ReturnAuthorization") }
     scope :voucher, -> { where(originator_type: "Voucher") }
+    scope :voucher_tax, -> { where(originator_type: "Voucher").where("label LIKE 'Tax%'") }
```

Comment:
That should work for now but it's not ideal. I'm not sure what the ideal solution would be though. Maybe if we created voucher tax adjustments for each tax adjustment? Then we knew that it's voucher tax because it's coming from a voucher and adjusting a tax adjustment. Our of scope here though.

Reply:
If this is a problem going forward then we should fix it soon though. There's also the issue of multiple tax rates, some inclusive and some exclusive. Related but doesn't necessarily require splitting the voucher tax adjustment... :thinking: